### ISSUE TITLE
Snap install: ncspot and urxvt

### ISSUE THREAD

**Issue by dboneham** [hrkfdn/ncspot#139]
The program does not load when using urxvt, which happens to be my preferred terminal. No problems if I load using xfce terminal or xterm.

Error message is: Error opening terminal: rxvt-256color.

**Comment by benys** [comment]
Terminal st shows the same problem - Error opening terminal: st-256color.

**Comment by hrkfdn** [comment]
Hi, unfortunately I don't maintain the snap package so I can't help you out with it. I'm sorry!

**Comment by dboneham** [comment]
Thanks. No problem. I'm going to try a different way. Shame becuase
snaps are very easy for lazy users comme moi.

**Comment by 0atman** [comment]
For anyone seeing this, I installed today:

Snap package - no sound
`cargo build ncspot` - works perfectly.

Fix for ubuntu: install the deps in the readme, run `cargo build ncspot`. Then run `~/.cargo/bin/ncspot`

**Comment by markstos** [comment]
@dboneham @benys I also ran into the `error opening terminal` error with snap, trying to use `alacritty` as the terminal. I think we can rule out the terminal being a problem since it happend with three different ones. 

I was able to fix it by setting an explicit TERM value when launching it:

     alacritty -e env TERM=screen-256color ncspot

My guess is that `ncspot` is running inside a container which does have access to custom "terminfo" entries that exist outside the container.

I suspect this can be fixed by explicitly setting the TERM environment variable within the `snap`.

I'll continue the conversation in https://github.com/popey/ncspot-snap/issues/1 which is a parallel bug report in the repo that contains the snap packaging details.
